<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskOneWidget</class>
 <widget class="QWidget" name="TaskOneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="font">
      <font>
       <family>黑体</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,1">
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <widget class="QWidget" name="buttonWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="openButton">
           <property name="font">
            <font>
             <family>黑体</family>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="text">
            <string>打开正则表达式文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveRegexButton">
           <property name="text">
            <string>保存正则表达式文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="uploadCodeButton">
           <property name="font">
            <font>
             <family>黑体</family>
            </font>
           </property>
           <property name="text">
            <string>上传待分词源代码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReButton">
           <property name="font">
            <font>
             <family>黑体</family>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>正则表达式分析</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,15">
      <item>
       <widget class="QWidget" name="selectWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>黑体</family>
            </font>
           </property>
           <property name="text">
            <string> 请选择你希望展示的正则表达式:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>黑体</family>
         </font>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="nfaTab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="nfaTableWidget">
            <property name="font">
             <font>
              <family>黑体</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="dfaTab">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableWidget" name="dfaTableWidget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="miniDfaTab">
         <attribute name="title">
          <string>页</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTableWidget" name="miniDfaTableWidget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="codeTab">
         <attribute name="title">
          <string>页</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTextEdit" name="codeView">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="lexTab">
         <attribute name="title">
          <string>页</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTextEdit" name="srcEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="lexButton">
            <property name="font">
             <font>
              <family>黑体</family>
             </font>
            </property>
            <property name="text">
             <string>源代码分词</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="resultTableWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
